pkg/infra/usagestats/service/service.go

```diff
@@ -110,12 +96,8 @@ func (uss *UsageStats) Run(ctx context.Context) error {
 				nextSendInterval = sendInterval
 				sendReportTicker.Reset(nextSendInterval)
 			}
-
-			// always reset live stats every report tick
-			uss.resetLiveStats()
```

Comment:
Noticed that we don't have live stats reset after sending in new code. Is this intentionally?

Reply:
Hmm.. I'm not sure about that. 

I'd say the reset is still present but in a different place (look [here](https://github.com/grafana/grafana/blob/e0e1e9a6481df55b6eb9504fda5efa53aa0a8011/pkg/services/live/live.go#L1055)).

Does that make sense? 🤔  Thanks! 🙏🏻

Reply:
I see, thanks!